src/lib/import-export/import/handlers/backup-handler-factory.ts

```diff
@@ -8,6 +8,16 @@ export interface BackupHandler {
 	extractFiles( file: BackupArchiveInfo, extractionDirectory: string ): Promise< void >;
 }
 
+const EXCLUDED_FILES_PATTERNS = [
+	/^__MACOSX\/.*/, // MacOS meta folder
```

Comment:
Should we exclude `/^\.DS_Store$/` too ? Or is that already covered by `/^\..*/, // Unix hidden files at root`?

Reply:
Good question. As you pointed out, we don't need to explicitly exclude `.DS_Store` as it will be  covered by that pattern. In fact, the compressed files I added in [backup-files.zip](https://github.com/user-attachments/files/16247647/backup-files.zip) contain that hidden folder and are not being processed.